symfony/plugins/orangehrmAdminPlugin/Service/SkillService.php

```diff
@@ -0,0 +1,173 @@
+<?php
```

Comment:
Seems this file contains unnecessary comments

Reply:
file removed